src/v/redpanda/admin_server.cc

```diff
@@ -121,6 +121,24 @@ admin_server::admin_server(
 ss::future<> admin_server::start() {
     configure_metrics_route();
     configure_admin_routes();
+    _server._routes.register_exeption_handler(
```

Comment:
Rather than doing this via the exception handler hook (which can only see the exception object), this could be done in side `register_route` and `register_route_raw`, where we have access to the request and auth state.

That way when logging an error, we can log:
- Client address
- URL path
- Username, if authenticated.

Reply:
implemented